Emma/emma_libs/memoryManager.py

```diff
@@ -184,6 +195,157 @@ def createStandardReports():
         #     graph.node('C', 'C', _attributes={'shape': 'triangle'})
         #
         #     print(graph.source)
+        def createSvgReport(startPoint, endPoint, scalingValue="1"):
```

Comment:
Please add x/y scaling: https://svgwrite.readthedocs.io/en/latest/classes/mixins.html#svgwrite.mixins.Transform.scale

The command line arguement would look like: `--scaling=1:2` (x:y).